### PR TITLE
Feature/jose/UI converter

### PR DESCRIPTION
**Pull request by Git-Tonio**

### CODE REVIEW

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 1]
```diff
@@ -1,94 +1,177 @@
...
-    static JLabel l, l1, l3, l4;
-
-    
-    // combobox
-    static JComboBox c1, c2;
     private ConvertController controller;
+    private FormatConvertController formatController;
+    private String path;
+
+    private JComboBox<String> combotype;
```
Comment (sergioale9) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: The name of the variable must respect the established conventions

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java [hunk 2]
```diff
@@ -1,94 +1,177 @@
...
+    private JComboBox<String> comboformat;
```
Comment (sergioale9) [inline app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: The name of the variable must respect the established conventions

**Review by sergioale9 — CHANGES_REQUESTED** [review]
Please review the comments before merging this branch

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java (hunk 1 above)
Comment (Git-Tonio) [reply app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: the variables have already been renamed

File: app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java (hunk 2 above)
Comment (Git-Tonio) [reply app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java]: the variables have already been renamed

**Review by sergioale9 — APPROVED** [review]
looks good, approved

**Pull request merged**